scss/_maps.scss

```diff
@@ -6,6 +6,39 @@
 $theme-colors-rgb: map-loop($theme-colors, to-rgb, "$value") !default;
 // scss-docs-end theme-colors-rgb
 
+$theme-colors-text: (
```

Comment:
I've only scratched the surface of this PR so far but there's something here that can be disturbing.
In light mode:

* `color: var(--bs-primary-text)` will make the text dark blue
* `.text-primary` will make the text blue
* `.text-primary-emphasis` whill make the text dark blue.

Just be reading the CSS vars and Sass vars, I would expect that `var(--bs-primary-text)` would be the same color as `.text-primary` rather than `.text-primary-emphasis`.

But again, I'm just at the beginning of the review so maybe that's normal.

Reply:
So the color stuff _is_ confusing right now. Here's the deal:

Right now, we assume all our default/`500` colors should meet minimum contrast levels. They mostly do, but not all of them. Like we can't use `$cyan-500` as text against our white background. As such, we need a _darker_ color for that text. That's the purpose of the new `{themeColor}-text` colors as opposed to the `{themeColor}` colors. I _think_ v6 will deprecate the existing `.text-{themeColor}` classes for the `.text-{themeColor}-text` classes, but not 100% sure. These new text colors all provide high enough contrast in light and dark mode against default background colors and the new subtle theme background colors.

Alternatively we could go the Tailwind route and make _every_ tint and shade of every color available, but that feels like far too much.

Reply:
So the color stuff is confusing right now. Here's the deal:

Right now, we assume all our default/500 colors should meet minimum contrast levels. They mostly do, but not all of them. Like we can't use $cyan-500 as text against our white background. As such, we need a darker color for that text. That's the purpose of the new {themeColor}-text colors as opposed to the {themeColor} colors. I think v6 will deprecate the existing .text-{themeColor} classes for the .text-{themeColor}-text classes, but not 100% sure. These new text colors all provide high enough contrast in light and dark mode against default background colors and the new subtle theme background colors.

Alternatively we could go the Tailwind route and make every tint and shade of every color available, but that feels like far too much.

Reply:
Seems fair. Thanks for the clarification. Resolving it and continuing the global review :)